app/src/main/java/org/simple/clinic/storage/migrations/Migration_80.kt

```diff
@@ -0,0 +1,16 @@
+package org.simple.clinic.storage.migrations
+
+import androidx.room.migration.Migration
+import androidx.sqlite.db.SupportSQLiteDatabase
+import org.simple.clinic.storage.inTransaction
+import javax.inject.Inject
+
+@Suppress("ClassName")
+class Migration_80 @Inject constructor(): Migration(79, 80) {
+
+  override fun migrate(database: SupportSQLiteDatabase) {
+    database.inTransaction {
+      execSQL(""" ALTER TABLE "Facility" ADD COLUMN "syncGroup" TEXT """)
```

Comment:
Should we add a default value or will it be done automatically during migration?

Reply:
The default value is `NULL`, which will be set automatically since the field is nullable. A later PR will trigger a resync so that the sync groups are fetched from the server.